apps/dashboard/app/models/posix_file.rb

```diff
@@ -151,7 +151,13 @@ def handle_upload(tempfile)
     FileUtils.mv tempfile, path.to_s
     File.chmod(mode, path.to_s)
 
-    path.chown(nil, path.parent.stat.gid) if path.parent.setgid?
+    begin
+      path.chown(nil, path.parent.stat.gid) if path.parent.setgid?
+    rescue StandardError => e
+      Rails.logger.info("cannot chown #{path} because of error: #{e}")
+    end
+
+    nil
```

Comment:
I'm also ensuring we return nil here because other handle_upload methods return a Transfer.